tests/EnqueueMessageProcessorTest.php

```diff
@@ -0,0 +1,210 @@
+<?php
+/**
+ * This file is part of the prooph/psb-enqueue-producer.
+ * (c) 2017-2017 prooph software GmbH <contact@prooph.de>
+ * (c) 2017-2017 Sascha-Oliver Prolic <saschaprolic@googlemail.com>
+ * (c) 2017-2017 Maksym Kotliar <kotlyar.maksim@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ProophTest\ServiceBus\Enqueue;
+
+use Enqueue\Consumption\Result;
+use Enqueue\Null\NullContext;
+use Enqueue\Null\NullMessage;
+use Enqueue\Psr\PsrMessage;
+use Enqueue\Psr\PsrProcessor;
+use PHPUnit\Framework\TestCase;
+use Prooph\Common\Messaging\DomainMessage;
+use Prooph\Common\Messaging\PayloadTrait;
+use Prooph\ServiceBus\CommandBus;
+use Prooph\ServiceBus\EventBus;
+use Prooph\ServiceBus\Message\Enqueue\EnqueueMessageProcessor;
+use Prooph\ServiceBus\Message\Enqueue\EnqueueSerializer;
+use Prooph\ServiceBus\QueryBus;
+use ProophTest\ServiceBus\Mock\DoSomething;
+use ProophTest\ServiceBus\Mock\FetchSomething;
+use ProophTest\ServiceBus\Mock\SomethingDone;
+use React\Promise\Promise;
+
+final class EnqueueMessageProcessorTest extends TestCase
+{
+    public function testShouldImplementPsrProcessorInterface()
```

Comment:
first test are defined like this:

```
    /**
     * @test
     */
    public function it_tests_whatever
```

and here you have `testShouldImplementPsrProcessorInterface` - can we have one style for all tests? I prefer the first option, as we use this in all prooph components, and it's easier to read.